src/main/java/com/puppycrawl/tools/checkstyle/filters/JavaPatchFilterElement.java

```diff
@@ -53,6 +53,10 @@
     static {
         CHECK_TO_ANCESTOR_NODES_MAP.put("AvoidNestedBlocks",
                 Arrays.asList(TokenTypes.SLIST));
+        CHECK_TO_ANCESTOR_NODES_MAP.put("DefaultComesLast",
+                Arrays.asList(TokenTypes.METHOD_DEF));
```

Comment:
METHOD_DEF is too big scope. You should use `LITERAL_SWITCH`.
AST example http://rveach.no-ip.org/checkstyle/checkstyle.php?action=view&config=72fba57f86eaf7df54f418b381c28422&code=f170111209d932f24709c0387d4c787f&checkstyle=checkstyle-8.35-all.jar&printTree=true&printSuppressions=

Reply:
done, I agree.